prototyping/Search.py

```diff
@@ -0,0 +1,50 @@
+from MockArticle import Article
+
+class Search:
+    """
```

Comment:
Hmm, with this set up, it looks like you have to pass all existing articles with every query, which seems a bit inefficient. Maybe this should be a bit more multipurpose, like a SearchEngine class?

For a SearchEngine class, I would expect more like:
```python
def __init__(self, articles, settings):
   ...

def search(self, query):
   ...
```